### DISCUSSION TITLE
Starting iOS & android at the same time

### DISCUSSION THREAD

**Discussion by Pastajello** [dotnet/maui#15499] [General]
Hello,

I'm currently moving my project from XF to MAUI. I have smaller and bigger problems, but I'm not sure about one thing and can't really wrap my head how to solve it. And it's about an ability to sun both platform at the same time.

With Xamarin.Forms we can set 2 project to run at the same time, work with hotrelaod and see how design changes are reflected on both iOS and Android. 

Is this thing possible in MAUI?

**Comment by HausBJB** [comment]
Hi
I had written about the problem so many times in the blog. But I think this feature is not used by many, so it is not missed
